pkg/agent/openflow/network_policy.go

```diff
@@ -2066,6 +2066,12 @@ type featureNetworkPolicy struct {
 	// egressTables map records all IDs of tables related to egress rules.
 	egressTables map[uint8]struct{}
 
+	// loggingGroupCache is a storage for the logging groups, each one includes 2 buckets: one is to send the packet
+	// to antrea-agent using packetIn mechanism, the other is to enforce the packet in the OVS pipeline to continue
+	// forwarding logic. The key is the next table used in the second bucket, and the value is the Openflow group.
```

Comment:
"to force the packet to continue forwarding in the OVS pipeline"?

Reply:
updated.